atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/CqlMetadata.java

```diff
@@ -76,8 +76,10 @@ public LightweightOppToken newToken(ByteBuffer byteBuffer) {
         }
     }
 
-    public Set<Host> getReplicas(String keyspace, Range<LightweightOppToken> range) {
-        return metadata.getReplicas(quotedKeyspace(keyspace), toTokenRange(range));
+    public Set<InetSocketAddress> getReplicas(String keyspace, Range<LightweightOppToken> range) {
+        return metadata.getReplicas(quotedKeyspace(keyspace), toTokenRange(range)).stream()
+                .map(host -> host.getEndPoint().resolve())
```

Comment:
I think the answer is no, but just to double-check: this doesn't do any kind of DNS loookup?

Reply:
No - the EndPoint implementation used is a TranslatedAddressEndPoint, which just wraps the InetSocketAddress that we gave it when configuring the cluster.